includes/definitions/discovery/arubaos.yaml

```diff
@@ -0,0 +1,12 @@
+mib: 'ARUBA-MIB'
+modules:
+    processors:
+          data:
+              -
+                  oid: WLSX-SYSTEMEXT-MIB::sysExtProcessorLoad
```

Comment:
Is this a table as the supervisor card cpu one below would also get picked up in this.

Reply:
I was trying to do this to override the description for the .1 OID.  Maybe there's a better way?

walk of : .1.3.6.1.4.1.14823.2.2.1.2.1.13.1
.1.3.6.1.4.1.14823.2.2.1.2.1.13.1.2.1 = STRING: Supervisor Card CPU
.1.3.6.1.4.1.14823.2.2.1.2.1.13.1.2.8 = STRING: Network Processor CPU8
.1.3.6.1.4.1.14823.2.2.1.2.1.13.1.2.9 = STRING: Network Processor CPU9
.1.3.6.1.4.1.14823.2.2.1.2.1.13.1.2.10 = STRING: Network Processor CPU10
.1.3.6.1.4.1.14823.2.2.1.2.1.13.1.2.11 = STRING: Network Processor CPU11
.1.3.6.1.4.1.14823.2.2.1.2.1.13.1.2.12 = STRING: Network Processor CPU12
.1.3.6.1.4.1.14823.2.2.1.2.1.13.1.2.13 = STRING: Network Processor CPU13
.1.3.6.1.4.1.14823.2.2.1.2.1.13.1.2.14 = STRING: Network Processor CPU14
.1.3.6.1.4.1.14823.2.2.1.2.1.13.1.2.15 = STRING: Network Processor CPU15
.1.3.6.1.4.1.14823.2.2.1.2.1.13.1.3.1 = INTEGER: 2
.1.3.6.1.4.1.14823.2.2.1.2.1.13.1.3.8 = INTEGER: 0
.1.3.6.1.4.1.14823.2.2.1.2.1.13.1.3.9 = INTEGER: 1
.1.3.6.1.4.1.14823.2.2.1.2.1.13.1.3.10 = INTEGER: 0
.1.3.6.1.4.1.14823.2.2.1.2.1.13.1.3.11 = INTEGER: 0
.1.3.6.1.4.1.14823.2.2.1.2.1.13.1.3.12 = INTEGER: 0
.1.3.6.1.4.1.14823.2.2.1.2.1.13.1.3.13 = INTEGER: 0
.1.3.6.1.4.1.14823.2.2.1.2.1.13.1.3.14 = INTEGER: 0
.1.3.6.1.4.1.14823.2.2.1.2.1.13.1.3.15 = INTEGER: 0

.1 always seems to be Supervisor Card CPU and the rest are Network Processor CPU{{ $index }}.

Reply:
Does it override the supervisor card cpu as you expect?

If so let's leave it at that.

You need to move the `WLSX-SYSTEMEXT-MIB` to the `mib:` line at the top: `ARUBA-MIB:WLSX-SYSTEMEXT-MIB`

Reply:
Yeah, override looks good on the graphs...
![image](https://user-images.githubusercontent.com/5092581/38650412-d8e49da0-3e4f-11e8-8a77-06a1b58dee3a.png)

Reply:
We'll go with that for now. Only way round it would be to add an option to discard based on indexes. However we shall see if we need to cross that bridge when more devices need it.